cfy_manager/components/nginx/scripts/sepolicy_deploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
```

Comment:
I suggest we move this into a bunch of calls in python, or at the very least set -e, whatever the one is for failing on unset vars, and set -o pipefail (if I remembered that command correctly)

Reply:
Good point, I'll fix it